plugins/org.wso2.developerstudio.eclipse.esb.libraries/META-INF/MANIFEST.MF

```diff
@@ -117,7 +115,9 @@ Bundle-ClassPath: .,
  lib/org.wso2.carbon.rule.backend_4.4.9.jar,
  lib/org.wso2.carbon.rule.common_4.4.9.jar,
  lib/org.wso2.carbon.rule.kernel_4.4.9.jar,
- lib/org.wso2.carbon.rule.mediation_4.4.9.jar
+ lib/org.wso2.carbon.rule.mediation_4.4.9.jar,
+ lib/synapse-extensions_2.1.7.wso2v80.jar,
```

Comment:
the core version is wso2v82 but the extensions version is wso2v80, is this intentional?

Reply:
Yeah, this was intentional. We have updated only the synapse-core v82 (we don't have a requirement to update the synapse-extension version at this point).